mm2src/coins/tendermint/tendermint_tx_history_v2.rs

```diff
@@ -38,6 +38,7 @@ const IBC_SEND_EVENT: &str = "ibc_transfer";
 const IBC_RECEIVE_EVENT: &str = "fungible_token_packet";
 const IBC_NFT_RECEIVE_EVENT: &str = "non_fungible_token_packet";
 const DELEGATE_EVENT: &str = "delegate";
+const UNDELEGATE_EVENT: &str = "unbond";
```

Comment:
Isn't this a typo by a chance (not 'unbound')?

Reply:
[No it's not](https://docs.cosmos.network/v0.46/modules/staking/02_state_transitions.html)